docs/USAGE.md

```diff
@@ -73,7 +84,7 @@ or
 
 **OPENVINO_TF_CONVERT_VARIABLES_TO_CONSTANTS**
 
-This variable is enabled by default, and it freezes variables from TensorFlow's ReadVariableOp as constants during the graph translation phase. Highly recommended to keep it enabled to ensure optimal inference latencies on eagerly executed models. Disable it when model weights are modified after loading the model for inference.
+This variable is disabled by default, and it freezes variables from TensorFlow's ReadVariableOp as constants during the graph translation phase. Highly recommended to enable it to ensure optimal inference latencies on eagerly executed models. Disable it when model weights are modified after loading the model for inference.
 
```

Comment:
Is this applicable to specific types of TF 2.x models like Keras/h5 format only or is it applicable to saved model as well? Will saved model not convert the variables to constants automatically?

Reply:
ReadVariableOp API is part of TF 1.x too, and it could be present in TF 1.x models too. Saved model: https://www.tensorflow.org/guide/saved_model is supported by TF2.x too,.